spec/lib/reports/payments/itemised_payment_totals_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+module Reporting
+  module Reports
+    module Payments
+      describe ItemisedPaymentTotals do
+        context "As a site admin" do
+          let(:user) do
+            user = create(:user)
+            user.spree_roles << Spree::Role.find_or_create_by!(name: 'admin')
+            user
+          end
+          subject do
+            Base.new user, {}
+          end
+
+          let!(:distributor) { create(:distributor_enterprise) }
+
+          let!(:order) do
+            create(:completed_order_with_totals, line_items_count: 1, distributor: distributor)
+          end
+
+          let(:current_user) { distributor.owner }
+          let(:params) { { display_summary_row: false } }
+          let(:report) do
+            ItemisedPaymentTotals.new(current_user, params)
+          end
+
+          let(:table_headers) do
+            report.table_headers
+          end
+
+          let(:report_table) do
+            report.table_rows
+          end
+
+          it "generates the report" do
+            expect(report_table.length).to eq(1)
+          end
+
+          it "Should return headers" do
+            expect(report.table_headers).to eq([
+                                                 "Payment State",
+                                                 "Distributor",
+                                                 "Product Total ($)",
+                                                 "Shipping Total ($)",
+                                                 "Outstanding Balance ($)",
+                                                 "Total ($)"
+                                               ])
+          end
+          it "translates payment_states" do
+            first_row_first_column_value = report_table.first.first
+            translated_payment_state = i18n_translate("balance_due")
+            expect(translated_payment_state).to eq first_row_first_column_value
```

Comment:
Usually, we don't translate things in the specs. I think here we can simply use:
```
expect(first_row_first_column_value).to eq "Balance due". 
```


(I assume this was previously `balance_due`, right?)

Reply:
@jibees Thank you for the feedback!

Is it possible to add in a similar workaround for the translation function in `payment_reports_spec.rb`? We are unable to run the spec locally so we are unsure what `order.payment_state` represents. Is it also “balance due”?

Reply:
Some lines bellow, `order` is defined like this:

https://github.com/openfoodfoundation/openfoodnetwork/blob/d61affd82be0db5ee91e542153d32748322a0c59/spec/system/admin/reports/payments_report_spec.rb#L8-L15

So I guess the state should be `Complete`. 

I'm wondering why you're not able to run the spec locally? Do you have a particular error?